content/en/docs/ambient/install/istioctl/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Install with istioctl
+description: Install Istio with support for ambient mode using the istioctl command line tool.
+weight: 10
+keywords: [istioctl,ambient]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< tip >}}
+Follow this guide to install and configure an Istio mesh with support for ambient mode.
+If you are new to Istio, and just want to try it out, follow the
+[quick start instructions](/docs/ambient/getting-started) instead.
+{{< /tip >}}
+
+This installation guide uses the [istioctl](/docs/reference/commands/istioctl/) command-line
+tool to provide rich customization of Istio installation options. It has user input validation
+to help prevent installation errors and customization options to override any aspect of the
+configuration.
+
+Using these instructions, you can select any one of Istio's built-in
+[configuration profiles](/docs/setup/additional-setup/config-profiles/)
+and then further customize the configuration for your specific needs.
+
+The `istioctl` command supports the full [`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/)
+via command-line options for individual settings or for passing a yaml file containing an `IstioOperator`
+{{<gloss CRDs>}}custom resource (CR){{</gloss>}}.
+
+## Prerequisites
+
+Before you begin, check the following prerequisites:
+
+1. [Download the Istio release](/docs/setup/additional-setup/download-istio-release/).
```

Comment:
1. I don't think we should tell people to "download the Istio release" - we should probably just tell them to 

1. Install `istioctl` (which is currently buried -> https://istio.io/latest/docs/ops/diagnostic-tools/istioctl/#install-hahahugoshortcode686s2hbhb - we should maybe promote that from `diagnostic-tool-only setup` to `proper istioctl setup`.

2. Document the install from that. `istioctl` binary by itself should probably be enough to bootstrap, if the goal here is "istioctl is the simple, single entry point".

Reply:
This is a direct lift from the sidecar `install with istioctl page.

The "download release" page was separated out from the sidecar getting started docs, in order to document the options without putting them in front of people on day 1.  It's a weird wording but remember "Istio release" is "istioctl plus samples".  We use it throughout the docs at present.

We could rename it "download client" and condense these two pages?

(and maybe even tell people you can use `brew` or apt/rpm)

Reply:
Can we just tell people to install `istioctl` here using the same instructions as the "install Istioctl page"?

Maybe even link to that page for further context/

https://preliminary.istio.io/latest/docs/ops/diagnostic-tools/istioctl/#install-hahahugoshortcode708s2hbhb

That is, the script that doesn't require downloading the samples.

We should be asking people to download the samples on the page where they actually need the samples, which is not this page.

And we should be using the same install script consistently everywhere.

Reply:
https://github.com/istio/istio.io/issues/15585